Prestige cheats are in! And a QoL zoom change =)

Kurdran opened the pull request:
Turned the zoom multiplier into a float, for more flexibility

Added support for (most) Prestige Difficulties!
Some difficulties are not included, their models were non-trivial:
- All the non-prestige ones
- P1, P3, P11, P16, P19, P20

Kurdran committed:
Turned the zoom multiplier into a float, for more flexibility

Added support for Prestige Difficulties!
- Some difficulties are not included, their models were non-trivial:
    - All the non-prestige ones
    - P1, P3, P11, P16, P19, P20

Josiwe commented on Josiwe.ATS.Cheats.Config.json:
```diff
@@ -17,5 +17,21 @@
-}
+    "CornerstonePicksPerSeason": 1,
+
+    "P4Amount": 10,
```
Since we have no way to create in-game dialogues, it might be helpful here to add something like this:

"P4Comment": "Villagers leave twice as fast when at low resolve. Set to 10 for the default behavior, other values work like this... etc"

Josiwe replied:
Actually isn't this one supposed to be p3?

Kurdran replied:
Good catch, yeah it should be 3

Kurdran replied:
No, 4 is blueprint rerolls costing 10 amber
![Screenshot 2023-12-02 202209](https://github.com/Josiwe/Josiwe.ATS.Cheats/assets/18384434/2c6f4905-8332-4f23-9fd6-f122d95149c6)

Josiwe reviewed (commented):
A couple of thoughts on this.

1) You've put in way more work than me at this point. I don't mind doing code review and publishing releases, but would you be more comfortable forking it and taking ownership? You could call it Kurdran's mod and just list me as a contributor. 

2) I think one thing you could do with this is add a sample config file that focuses on disabling the cheats and upping the difficulty. "INSANE P20 DIFFICULTY" seems like the kind of thing the content creator community would go nuts over. Someone like [PotatoMcWhiskey](https://www.youtube.com/channel/UCAF_yy-Y4yl4VMiQ7RV3STw) might really enjoy playing with super hard mode.

Kurdran commented:
On 1: I don't know for how long I can do this. I'll be travelling for work next week and I have no return date set yet.
On 2: That's a great idea, maybe more doable now with the prestige cheats, but point 1 again =(
Edit: I created the fork though, any new PR's will come from there
> A couple of thoughts on this.
> 
>     1. You've put in way more work than me at this point. I don't mind doing code review and publishing releases, but would you be more comfortable forking it and taking ownership? You could call it Kurdran's mod and just list me as a contributor.
> 
>     2. I think one thing you could do with this is add a sample config file that focuses on disabling the cheats and upping the difficulty. "INSANE P20 DIFFICULTY" seems like the kind of thing the content creator community would go nuts over. Someone like [PotatoMcWhiskey](https://www.youtube.com/channel/UCAF_yy-Y4yl4VMiQ7RV3STw) might really enjoy playing with super hard mode.

Kurdran committed:
Cleaned up a bit:
- Prestige variables have been reworked
- Added descriptions snatched from the game to each one

Added example config files for:
- ultimate fucking pussy mode (explorer)
- insane difficulty
- vanilla (for reference)

Kurdran committed:
Bug fixing!

Also updated the readme

Josiwe reviewed (approved):
Really great work here.
